### ISSUE TITLE
crash on uploading

### ISSUE THREAD

**Issue by Mo-** [avocode/avocode#720]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.5.3
Source UID: 126436
```
Error: invalid stored block lengths
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by vojtatranta** [comment]
Thank you for your report.
Could you please send your folder ~/.avcd to petr@avocode.com?
We know about this bug and we are now testing our solution to this.

**Comment by petrbrzek** [comment]
Hi, we've released 2.5.4 that should fix this issue. If you still have this problem in the new app, feel free to report it as a new issue.
